src/clojure/clojang/jinterface/otp/messaging.clj

```diff
@@ -153,20 +215,33 @@
   (get-pid [this]
     (.self this))
   (send
-    ([this recip-pid-or-name msg]
-      (.send this recip-pid-or-name msg))
+    ([this ^OtpErlangPid recip-pid ^OtpErlangObject msg]
+     (.send this recip-pid msg))
     ([this mbox-name node-name msg]
-      (.send this mbox-name node-name msg)))
+     (.send this mbox-name node-name msg)))
   (!
-    ([this recip-pid-or-name msg]
-      (.send this recip-pid-or-name msg))
+    ([this recip-pid msg]
+     (send this recip-pid msg))
```

Comment:
I've defined `!` in terms of `send`, for simplicity. Note that the same [caveats](https://github.com/oubiwann/clojang/pull/20/files#r51930297) apply here.